features/step_definitions/import_users.rb

```diff
@@ -0,0 +1,55 @@
+require 'open3'
+
+Given 'there are no users in the database' do
+  # NOOP
+end
+
+Given 'there are users in the database' do
+  User.create!(name: 'Alixe', email: 'alice@example.com')
```

Comment:
I'm adding FactoryGirl for the objectives features, but we can unify that later.